source/portable/NetworkInterface/NXP1060/NetworkInterface.c

```diff
@@ -0,0 +1,810 @@
+                                                NULL,
+                                                configMAX_PRIORITIES - 1,
+                                                &receiveTaskHandle );
+
+                    if( ( receiveTaskHandle == NULL ) || ( xTaskCreated != pdPASS ) )
+                    {
+                        FreeRTOS_printf( ( "Failed to create the handler task." ) );
+                        break;
+                    }
+
+                    /* Enable the interrupt and set its priority to the minimum
+                     * interrupt priority.  */
+                    NVIC_SetPriority( ENET_IRQn, ENET_PRIORITY );
+                    NVIC_EnableIRQ( ENET_IRQn );
+
+                    eEMACState = xEMAC_Ready;
+
+                    /* After this, the task should not be created. */
+                    xFirstCall = pdFALSE;
+                }
+                else
+                {
+                    break;
+                }
+            }
+            else
+            {
+                eEMACState = xEMAC_Ready;
+            }
+
+        /* Fall through. */
+        case xEMAC_Ready:
+            FreeRTOS_printf( ( "Driver ready for use." ) );
+
+            /* Kick the task once the driver is ready. */
+            if( receiveTaskHandle != NULL )
+            {
+                xTaskNotify( receiveTaskHandle, DRIVER_READY, eSetValueWithOverwrite );
+            }
+
+            xResult = pdPASS;
+
+            break;
+    }
+
+    return xResult;
+}
+
+BaseType_t xNetworkInterfaceOutput( NetworkBufferDescriptor_t * const pxNetworkBuffer,
+                                    BaseType_t xReleaseAfterSend )
+{
+    status_t result;
+    BaseType_t xReturn = pdFAIL;
+
+    if( bGlobalLinkStatus == true )
+    {
+        /* ENET_SendFrame copies the data before sending it. Therefore, the network buffer can
+         * be released without worrying about the buffer memory being used by the ENET_SendFrame
+         * function. */
+        result = ENET_SendFrame( ethernetifLocal->base,
+                                 &ethernetifLocal->handle,
+                                 pxNetworkBuffer->pucEthernetBuffer,
+                                 pxNetworkBuffer->xDataLength,
+                                 0,
+                                 false,
+                                 NULL );
+
+        switch( result )
+        {
+            case kStatus_ENET_TxFrameBusy:
+                FreeRTOS_printf( ( "Failed to send the frame - driver busy!" ) );
+                break;
+
+            case kStatus_Success:
+                iptraceNETWORK_INTERFACE_TRANSMIT();
+                xReturn = pdPASS;
+                break;
+        }
+    }
+
+    if( xReleaseAfterSend == pdTRUE )
+    {
+        vReleaseNetworkBufferAndDescriptor( pxNetworkBuffer );
+    }
+
+    return xReturn;
+}
+
+static void prvEMACHandlerTask( void * parameter )
+{
+    bool LinkUp = false;
```

Comment:
Nit-pick: Why not `bLinkUp`?

Reply:
Oops, makes sense, I will add this as a follow up PR.